src/decisionengine_modules/NERSC/sources/NerscSFApi.py

```diff
@@ -118,8 +125,13 @@ def requests_nersc(self, username):
 
     def send_query(self):
         results = []
+        print(self.constraints.get("usernames", []))
```

Comment:
Same comment as above - leftover print?

Reply:
yes leftover print statement that can be cleaned up.